{"uid":"355088f1e0a1b13a","name":"test_upgrade_from_edge","fullName":"tests.integration.ha_tests.test_upgrade#test_upgrade_from_edge","historyId":"fb5755da30904890c6dc32bb58026031","time":{"start":1716342235620,"stop":1716342889029,"duration":653409},"status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [idle] maintenance: refreshing the snap","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 849, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 883, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 256, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 182, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 172, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1772, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 195, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/tests/integration/ha_tests/test_upgrade.py\", line 99, in test_upgrade_from_edge\n    await ops_test.model.wait_for_idle(\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2956, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [idle] maintenance: refreshing the snap","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop","time":{"start":1716341227773,"stop":1716341227773,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1716341227774,"stop":1716341227920,"duration":146},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1716341227774,"stop":1716341227774,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1716342235619,"stop":1716342235619,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1716342235619,"stop":1716342235620,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [idle] maintenance: refreshing the snap","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 849, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 883, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 256, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 182, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 172, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1772, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 195, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/tests/integration/ha_tests/test_upgrade.py\", line 99, in test_upgrade_from_edge\n    await ops_test.model.wait_for_idle(\n  File \"/home/runner/work/test-runners-github-postgresql-operator/test-runners-github-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2956, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [idle] maintenance: refreshing the snap","steps":[],"attachments":[{"uid":"867191e8a1cad3e8","name":"log","source":"867191e8a1cad3e8.txt","type":"text/plain","size":5866}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"continuous_writes::finalizer","time":{"start":1716342889077,"stop":1716342890098,"duration":1021},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1716342890099,"stop":1716342890099,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1716342890103,"stop":1716342890557,"duration":454},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1716342890557,"stop":1716342890558,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1716342890558,"stop":1716342890558,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1716342890558,"stop":1716342890558,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_upgrade"},{"name":"host","value":"fv-az1114-972"},{"name":"thread","value":"7218-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_upgrade"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":20,"skipped":0,"passed":20,"unknown":0,"total":40},"items":[{"uid":"6b41852cf5ab5f89","reportUrl":"../53//#testresult/6b41852cf5ab5f89","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [executing] waiting: waiting for database initialisation\n  postgresql/2 [idle] waiting: other units upgrading first...","time":{"start":1716255775636,"stop":1716256426305,"duration":650669}},{"uid":"4245dfaf3112ed1f","reportUrl":"../52//#testresult/4245dfaf3112ed1f","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [idle] waiting: waiting for database initialisation","time":{"start":1716170155343,"stop":1716170804997,"duration":649654}},{"uid":"f1a07bc63dbdad21","reportUrl":"../51//#testresult/f1a07bc63dbdad21","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [executing] waiting: waiting for database initialisation","time":{"start":1716082943853,"stop":1716083591686,"duration":647833}},{"uid":"3ae703f84ae752e4","reportUrl":"../50//#testresult/3ae703f84ae752e4","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [idle] waiting: waiting for database initialisation","time":{"start":1715996726562,"stop":1715997372155,"duration":645593}},{"uid":"a6c63396b6fca1e7","reportUrl":"../49//#testresult/a6c63396b6fca1e7","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [executing] waiting: waiting for database initialisation","time":{"start":1715910315219,"stop":1715910960675,"duration":645456}},{"uid":"7f2efbdd337d8f0b","reportUrl":"../48//#testresult/7f2efbdd337d8f0b","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [idle] waiting: waiting for database initialisation","time":{"start":1715823538474,"stop":1715824187037,"duration":648563}},{"uid":"ad4c137eb828ae0f","reportUrl":"../47//#testresult/ad4c137eb828ae0f","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [executing] waiting: waiting for database initialisation\n  postgresql/2 [idle] waiting: other units upgrading first...","time":{"start":1715737419335,"stop":1715738065081,"duration":645746}},{"uid":"9bdd4809fc875ad3","reportUrl":"../46//#testresult/9bdd4809fc875ad3","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [executing] waiting: waiting for database initialisation","time":{"start":1715651016063,"stop":1715651666563,"duration":650500}},{"uid":"6fa9a1b4a4bcbc84","reportUrl":"../45//#testresult/6fa9a1b4a4bcbc84","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [executing] waiting: waiting for database initialisation","time":{"start":1715564557963,"stop":1715565205855,"duration":647892}},{"uid":"cfd1b7bc3a4e2a8b","reportUrl":"../44//#testresult/cfd1b7bc3a4e2a8b","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [idle] waiting: waiting for database initialisation","time":{"start":1715478403362,"stop":1715479054158,"duration":650796}},{"uid":"ed014bb52262618f","reportUrl":"../42//#testresult/ed014bb52262618f","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [executing] waiting: waiting for database initialisation\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [idle] waiting: other units upgrading first...","time":{"start":1715392210095,"stop":1715392856497,"duration":646402}},{"uid":"335ecdc83db66746","reportUrl":"../41//#testresult/335ecdc83db66746","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [executing] waiting: waiting for database initialisation","time":{"start":1715305402155,"stop":1715306048919,"duration":646764}},{"uid":"58b53054233c5263","reportUrl":"../40//#testresult/58b53054233c5263","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [executing] waiting: waiting for database initialisation\n  postgresql/2 [idle] waiting: other units upgrading first...","time":{"start":1715218845440,"stop":1715219492424,"duration":646984}},{"uid":"22cedfc76b1a84ad","reportUrl":"../39//#testresult/22cedfc76b1a84ad","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [executing] maintenance: refreshing the snap","time":{"start":1715132013615,"stop":1715132659801,"duration":646186}},{"uid":"f4f70d3755f34670","reportUrl":"../38//#testresult/f4f70d3755f34670","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [executing] waiting: waiting for database initialisation","time":{"start":1715046055902,"stop":1715046704331,"duration":648429}},{"uid":"7996b83afde67ddd","reportUrl":"../35//#testresult/7996b83afde67ddd","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: waiting for database initialisation\n  postgresql/1 [idle] active: \n  postgresql/2 [idle] waiting: other units upgrading first...","time":{"start":1714959641551,"stop":1714960287154,"duration":645603}},{"uid":"e18704296d69d11a","reportUrl":"../34//#testresult/e18704296d69d11a","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [executing] waiting: waiting for database initialisation\n  postgresql/2 [idle] waiting: other units upgrading first...","time":{"start":1714873975338,"stop":1714874622452,"duration":647114}},{"uid":"1c8fa37b63eb545b","reportUrl":"../33//#testresult/1c8fa37b63eb545b","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [idle] waiting: waiting for database initialisation","time":{"start":1714787252082,"stop":1714787900501,"duration":648419}},{"uid":"d6332f023695a692","reportUrl":"../32//#testresult/d6332f023695a692","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql/0 [idle] waiting: other units upgrading first...\n  postgresql/1 [idle] waiting: other units upgrading first...\n  postgresql/2 [idle] waiting: waiting for database initialisation","time":{"start":1714700628324,"stop":1714701275852,"duration":647528}},{"uid":"5f5cf426712b6a95","reportUrl":"../31//#testresult/5f5cf426712b6a95","status":"passed","time":{"start":1714614245316,"stop":1714614608127,"duration":362811}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"355088f1e0a1b13a.json","parameterValues":[]}